src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/condition-recommendations.mdx

```diff
@@ -0,0 +1,36 @@
+---
+title: Get condition recommendations
+metaDescription: For APM entities, you can see and create recommended alert conditions.
+watermark: Limited release
+---
+
+Use the New Relic Navigator's mini-overview to get alert condition recommendations for your monitored entities. With these recommendations, you can create alerts conditions to be notified when your entities aren't performing as they should.
+
+<Callout variant="caution">
+Condition recommendations are only available for APM-monitored entities.
+</Callout>
+
+## Why it matters
+
+Condition recommendations are all about making it easier for you to find entities and their conditions and thresholds you'd want to be notified about.
+
+Every recommendation includes one or more metrics that might interest you, as well as a threshold beyond which you might want to be notified.
+
+## Get condition recommendations
+
+From the entity detail pane, you can quickly create an alert condition. When you click **Add recommended conditions**, the UI provides a list of recommended conditions you might want to be notified about.
+
+Select one or more conditions and then assign them to an existing policy or create a new one.
+
+Once you've added a condition, you can change its settings like you can for any other NRQL condition.
+
+## How it works
```

Comment:
Maybe you could clarify that the more tags users add, the more the machine-learning algorithms work (if that's true)?

Reply:
I don't know that that's true.